### PR TITLE
fix(desktop): 聚合同一用户轮的用量明细

### PR DESCRIPTION
**Pull request by zqchris**
### 摘要

让 Desktop 费用 tooltip 在同一用户轮内聚合多个 Claude SDK segment 的 token、缓存和逐模型费用，避免只展示收尾 segment 而隐藏更早的子 Agent 消耗。

### 变更类型

- [ ] `feat` 新功能
- [x] `fix` 缺陷修复
- [ ] `refactor` / `perf` 重构或性能优化
- [ ] `docs` / `test` / `chore` 文档、测试或工程维护
- [ ] 其他：

### 范围

- 关联 Issue / 需求：用户反馈多段 Agent 任务的费用明细低于实际消耗
- 本 PR 包含：用户轮用量聚合、MessageActionBar 与 TodaySpendChip/QuotaHoverCard 接线、四语文案和回归测试
- 明确不包含：Codex descendant/subagent usage 采集、计价账本、模型价格、任务总额、服务端接口或历史重算
- 用户可见变化：同一用户请求跨自动续跑分段时，金额、token、缓存与按模型拆分现在都指向同一用户轮；不再暴露“最后 SDK 分段”的内部实现概念
- 是否存在 breaking change：无

### UI 变化

- 引用的设计规范：`docs/design-rules/DESIGN.md` §11 Voice & Content（面向用户只呈现一致的“本轮”语义）；§10 Light / Dark Dual-Mode Delivery Gate（复用现有语义 token 和组件样式，未新增颜色或单模式分支）

### 自动验证

```text
pnpm --filter desktop exec vitest run src/shared/__tests__/turnUsageDetails.test.ts src/renderer/lib/__tests__/userTurnUsage.test.ts src/renderer/components/chat/__tests__/MessageActionBar.test.tsx src/renderer/components/status/__tests__/QuotaHoverCard.test.tsx src/renderer/components/status/__tests__/TodaySpendChip.popover.test.tsx src/renderer/__tests__/todaySpendChip.test.ts
结果：6 files / 89 tests passed

pnpm check:i18n
结果：通过，四语 6716 个 key 一致；仅现有非阻断警告

pnpm check:i18n-glossary
结果：通过

pnpm --filter desktop typecheck
结果：通过

pnpm test:unit -- --workspace-concurrency=1
结果：通过，所有 required workspace 全绿

pnpm check:dco -- upstream/main..HEAD
结果：通过
```

### 手工验证

不涉及；本次通过 jsdom 组件测试验证 QuotaHoverCard 的用户轮金额、token 合计和逐模型费用。

### 未执行的验证

- 未启动 Desktop 做 Light / Dark 实机目检；本 PR 未改布局、颜色或样式，仅收口展示内容与数据投影

### 风险分类

- [ ] 无已知风险
- [ ] SQLite / migration
- [ ] system prompt
- [ ] 协议兼容
- [ ] 权限 / 安全 / 用户数据
- [ ] 存量插件兼容（批准状态 / 指纹 / manifest 校验 / 安装布局 / 包格式）
- [ ] 原生层 / fingerprint / OTA
- [ ] 跨平台差异
- [x] 其他：Renderer 展示聚合边界

### 影响与回滚

- 影响范围：Desktop 消息操作栏与 Claude 订阅用量卡的最近用户轮明细。历史加载如果从用户轮中段开始，保守回退为只展示目标 segment，不会误并到上一轮。
- 回滚 / 降级方式：回滚本 PR 即恢复收尾 segment 明细；计价账本和任务总额未变更。

### 提交前检查

- [x] 已 review 完整 diff
- [x] 每个 commit 都带 DCO 签名（`git commit -s`，见 [DCO](../DCO)）
- [x] UI 改动已在「UI 变化」注明引用的设计规范（不涉及 UI 则跳过）
- [x] 未提交凭证、令牌或授权文件
- [x] 已补充必要文档
- [x] 已确认测试结果或说明未执行原因

### CODE REVIEW

File: apps/desktop/src/shared/turnUsageDetails.ts [hunk 1]
```diff
@@ -107,6 +107,51 @@ function sanitizePerModelCost(
...
+    addModel(detail.model);
+    for (const model of detail.models ?? []) addModel(model);
+    for (const item of detail.perModelCost ?? []) {
+      const current = perModel.get(item.model);
+      perModel.set(
+        item.model,
+        current
+          ? (addCompatibleRegionalMoney([current, item.money], current.currency) ?? current)
+          : item.money,
+      );
```
[the remark this answers is not in the document]
Comment (zqchris) [reply apps/desktop/src/shared/turnUsageDetails.ts:140]: 已按建议修复：跨 segment 合并同一模型金额时不再把 current.currency 固定为偏好币种，现与 sanitizePerModelCost 共用默认账本币种选择规则，并在不兼容时保留新值。已新增同模型跨 segment 混币回归测试，覆盖默认账本币种应胜出的场景。

File: apps/desktop/src/renderer/i18n/locales/en/common.json [hunk 2]
```diff
@@ -5628,8 +5628,7 @@
       "turnCostEstimated": "Token value this turn (API-price equivalent)",
       "turnCostEstimatedValue": "Worth {{cost}}",
       "turnTokens": "{{tokens}} tokens",
-      "userTurnCostTotalLine": "User-turn total: {{cost}}",
-      "userTurnCostDetailsTitle": "Final SDK segment"
+      "userTurnCostDetailsTitle": "User-turn details"
```
[the remark this answers is not in the document]
Comment (zqchris) [reply apps/desktop/src/renderer/i18n/locales/en/common.json]: 已按术语边界修正用户可见文案：英文改为 Request details，日文改为 リクエストの明細，韩文改为 이번 요청 상세；zh-CN 的“本轮明细”保持不变。i18n 与 glossary 检查均已通过。

**Review by MagicLizi — APPROVED** [review]
Review passed. Clean refactor that correctly aggregates multi-segment usage details into a single user-turn view, with proper boundary detection, currency handling, edge-case fallbacks, and comprehensive tests.

**Comment by MagicLizi** [comment]
多 segment 用量散落在不同消息上一直让人困惑，聚合到同一轮后悬浮卡片终于能给出完整全景。边界检测和货币合并的边界情况也处理得干净。
